client/src/components/common/index.js

```diff
@@ -5,3 +5,4 @@ export { default as LabelItem } from './LabelItem';
 export { default as RowCol } from './RowCol';
 export { default as CardWithImage } from './CardWithImage';
 export { default as CustomModal } from './CustomModal';
+export { default as Form } from './Form';
```

Comment:
`export * as Form from './Form'`

will this work? I may be wrong, but give it a try once :)

Reply:
Below error occurres if we do this:-
![2019-12-13-1576236460_screenshot_1366x768](https://user-images.githubusercontent.com/30287271/70797116-d78a0480-1dc9-11ea-8caa-fefc28fe3200.jpg)